sdk/eventhub/eventhubs-checkpointstore-tables/tests.yml

```
@@ -0,0 +1,14 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/eventhubs-checkpointstore-blob"
```

Update blob to tables:
```suggestion
      PackageName: "@azure/eventhubs-checkpointstore-tables"
```